pkg/storaged/block/format-dialog.jsx

```diff
@@ -587,6 +612,17 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                         return client.blocks_fsys[path];
                 }
 
+                function block_swap_for_block(path) {
+                    if (keep_keys) {
+                        const content_block = client.blocks_cleartext[path];
+                        return client.blocks_swap[content_block.path];
+                    } else if (is_encrypted(vals))
+                        return (client.blocks_cleartext[path] &&
+                                client.blocks_swap[client.blocks_cleartext[path].path]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240117-111611-70e14048-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)